### PR TITLE
internal/monitoring: monitor number of identities

### PR DESCRIPTION
**Pull request by mhilton**

### CODE REVIEW

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-gce.jujugui.org:8080//job/candid/81/

File: store/mgostore/store.go [hunk 1]
```diff
@@ -244,3 +244,27 @@ func ensureIdentityIndexes(db *mgo.Database) error {
...
+
+var identityCountMapReduce = mgo.MapReduce{
+	Map:    `function() {p = this.providerid.split(':', 1); emit(p[0], p[1])}`,
+	Reduce: `function(key, values){ return values.length }`,
+}
+
+// IdentityCounts implements store.Store.IdentityCounts.
+func (s *identityStore) IdentityCounts(ctx context.Context) (map[string]int, error) {
+	coll := s.b.c(ctx, identitiesCollection)
+	defer coll.Database.Session.Close()
```
Comment (fabricematrat) [inline store/mgostore/store.go]: Why do we need to Close here ?

**Comment by mhilton** [comment]
:shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci-gce.theblues.io:8080/job/candid-merge

**Pull request merged**